packages/block-editor/src/components/block-settings-menu/block-settings-dropdown.js

```diff
@@ -168,6 +169,7 @@ export function BlockSettingsDropdown( {
 									/>
 								) }
 							</MenuGroup>
+							<ConvertToGroupButton onClose={ onClose } />
```

Comment:
Noting that `ConvertToGroupButton` internally uses `BlockSettingsMenuControls` fill that is going to render this menu item inside `BlockSettingsMenuControls.Slot` slot. So no matter where you put it in the rendering tree, it will get placed somewhere else. It's very confusing, to be honest :)

Reply:
Now that it's in the block-editor, we can remove the slot usage

Reply:
There are also reusable blocks that use it. I think this is why this slot was introduced in the first place. When searching for `<BlockSettingsMenuControls />` use I can also see other functionalities injected with it, with `PluginBlockSettingsMenuItem` exposed from `wp.editPost`.

Reply:
Yeah I don't think the slot should be removed. I'll check it today better.